src/client/mod.rs

```diff
@@ -336,7 +397,7 @@ impl WarpgrapherClient {
             "WarpGrapherClient read_rel -- query: {:#?}, input: {:#?}",
             query, input
         );
-        let result = graphql(self.endpoint.to_owned(), query, input.cloned())?;
+        let result = graphql(self.endpoint.to_owned(), query, input.cloned()).await?;
```

Comment:
If ownership is needed here, consider making the function signature take ownership and let the caller control cloning.